inst/WORDLIST

```diff
@@ -1,9 +1,32 @@
+args
+brushedPoints
+Closeable
+closeable
+desc
 Draggable
+draggable
+dt
 Forkers
+funder
+ggplot
+github
```

Comment:
@vedhav can you tick out `github` in package documentation so that we do not need to add this to WORDLIST?
Also, can you rename to `GitHub`?